cmd/utils/shared_config_options.go

```diff
@@ -279,6 +281,27 @@ func CrashTrackerTypeConfigOption(targetPointer interface{}) *config.ConfigOptio
 	}
 }
 
+func SchedulerConfigOptions(opts *scheduler.SchedulerOptions) []*config.ConfigOption {
+	return []*config.ConfigOption{
+		{
+			Name:        "scheduler-payment-job-seconds",
+			Usage:       "The interval in seconds for the payment jobs that synchronize transactions between SDP and TSS. 0 or negative values disable the job.",
+			OptType:     types.Int,
+			ConfigKey:   &opts.PaymentJobIntervalSeconds,
+			FlagDefault: 60 * 10, // 10 minutes
+			Required:    false,
+		},
+		{
+			Name:        "scheduler-receiver-invitation-job-seconds",
+			Usage:       "The interval in seconds for the receiver invitation job that sends invitations to new receivers. 0 or negative values disable the job.",
+			OptType:     types.Int,
+			ConfigKey:   &opts.ReceiverInvitationJobIntervalSeconds,
+			FlagDefault: 60 * 10, // 10 minutes
+			Required:    false,
+		},
+	}
```

Comment:
Isn't it possible to have a concurrence problem not disabling these jobs by default? Because we will have the events and the jobs.

Reply:
I'm going to disable these by default until we determine how safe it is to have both kafka and schedule jobs enabled.

Reply:
Addressed in c4ca6f0bf17d1f1a0b0f5d3a73bf967f3950fa99